scripts/install

```diff
@@ -1,12 +1,17 @@
-#!/bin/sh -e
+#!/usr/bin/env bash
```

Comment:
Do we need to be a bit wary here? Will linux users *consistently* have the bash shell available too, when running this script locally? What's the most widely available thing we can be doing here?

Reply:
I'd indeed prefer #666 to land beforehand so that

1. we keep sh
2. we remove all the os detection in the script

I introduced this change just because sh doesn't have the OSTYPE env var that is used to install the correct set of requirements on windows, but #666 should deal with that automatically

Reply:
Okay sure. I don't *quite* get it, but I'm happy to go along with that!

Final thing...

This change drops the `-e`. Are we able to still include the `-e` on this bash line, or do we need an explicit `set -e` in order to keep that behaviour?

Thanks so much for all your time on this! ✨

Reply:
as far as my limited shell knowledge goes the 2nd option `set -e` is what you need to do to exit on an error in bash but I can be wrong on this, I quickly looked at the man bash and didnt see a -e flag and when I tried to put it in the shebang it yelled at me:

```
❯ ./scripts/install
/usr/bin/env: ‘bash -e’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
   ~/PycharmProjects/uvicorn on    windows_action !1 ?1 ❯      

so I put a `set -e`
```